components/MnavBar.js

```diff
@@ -11,107 +48,45 @@ const Navbar = () => {
-                          />
-                        </svg>
-                      </button>
+            {setIsMenuOpen && (
+              <Menu isMenuOpen={isMenuOpen}>
+                <>
+                  <div className="relative">
+                    <div className={style.logo}>
+                      <Image src={WhiteLogo} alt="icon logo" />
                     </div>
                   </div>
-                  <nav>
-                    <ul className="space-y-4">
-                      <li>
-                        <a
-                          href="#"
-                          aria-label="Our product"
-                          title="Our product"
-                          className="font-medium tracking-wide text-amcovad-secondary-600 transition-colors duration-200"
-                        >
-                          Careers
-                        </a>
-                      </li>
-                      <li>
-                        <a
-                          href="#"
-                          aria-label="Our product"
-                          title="Our product"
-                          className="font-medium tracking-wide text-amcovad-secondary-600 transition-colors duration-200"
-                        >
-                          Products
-                        </a>
-                      </li>
-                      <li>
-                        <a
-                          href="#"
-                          aria-label="Product pricing"
-                          title="Product pricing"
-                          className="font-medium tracking-wide text-amcovad-secondary-600 transition-colors duration-200"
-                        >
-                          Services
-                        </a>
-                      </li>
-                      <li>
-                        <a
-                          href="#"
-                          aria-label="About us"
-                          title="About us"
-                          className="font-medium tracking-wide text-amcovad-secondary-600 transition-colors duration-200"
-                        >
-                          About us
-                        </a>
-                      </li>
-
-                      <li>
-                        <a href="#" className="" aria-label="Sign in" title="Sign in">
-                          <Button>Sign in</Button>
-                        </a>
-                      </li>
-                    </ul>
-                  </nav>
-                </div>
-              </div>
+                  <button
+                    aria-label="Close"
+                    className="absolute top-3 right-5 text-5xl text-amcovad-white cursor-pointer "
+                    onClick={() => setIsMenuOpen()}
+                  >
+                    &times;
+                  </button>
+                  <MenuContainer>
+                    <MenuItem href="/about">About </MenuItem>
```

Comment:
We should add this too to `data/menu.js` file to make it easier to update the menu in the future. I thought I added a comment in my last review.